packages/babel-preset-cozy-app/index.js

```diff
@@ -21,7 +21,7 @@ const presetEnvBrowserOptions = {
 
 const presetEnvNodeOptions = {
   targets: {
-    node: 12
+    node: 16
```

Comment:
This is not a chore commit anymore since : 
- we need this change to be deployed 
- this change will generate update in our production bundle.

Reply:
I'm using a fix commit then.